packages/Search/examples/bvh_driver/bvh_driver.cpp

```diff
@@ -25,8 +25,10 @@ std::vector<std::array<double, 3>>
 make_stuctured_cloud( double Lx, double Ly, double Lz, int nx, int ny, int nz )
 {
     std::vector<std::array<double, 3>> cloud( nx * ny * nz );
-    std::function<int( int, int, int )> ind = [nx, ny, nz](
-        int i, int j, int k ) { return i + j * nx + k * ( nx * ny ); };
+    std::function<int( int, int, int )> ind = [nx, ny, nz]( int i, int j,
+                                                            int k ) {
```

Comment:
Oh boy, this is quite ugly...

Reply:
so these are formatting changes from moving from 4.0 to 5.0?

Reply:
Yes.  All the changes to the C++ code in this PR correspond to applying the 5.0 new formatting style.